mcs_benchmark_data/pipelines/commonsense_qa/kagnet_commonsense_qa_submission_transformer.py

```diff
@@ -17,7 +20,15 @@ class KagnetCommonsenseQaSubmissionTransformer(_BenchmarkSubmissionTransformer):
 
     def transform(
         self,
+        *,
+        extracted_path: Path,
+        file_names: KagnetCommonsenseQaSubmissionFileNames,
         **kwds,
     ) -> Generator[_Model, None, None]:
 
-        yield from self._transform(submission_name="kagnet", **kwds)
+        yield from self._transform(

```

Comment:
If all of your public .transform implementations are just going to be a single upcall, you may as well make the _BenchmarkTransformer (or _BenchmarkSubmissionTransformer) .transform public and dispense with this.